README.md

```diff
@@ -376,9 +376,9 @@ These DataUnion-specific options can be given to `new StreamrClient` options:
 | Name                              | Returns             | Description                                                    |
 | :-------------------------------- | :------------------ | :------------------------------------------------------------- |
 | createSecret(\[name])             | string              | Create a secret for a Data Union                               |
-| setAdminFee(newFeeFraction)       | Transaction receipt | `newFeeFraction` is a `Number` between 0.0 and 1.0 (inclusive) |
 | addMembers(memberAddressList)     | Transaction receipt | Add members                                                    |
 | removeMembers(memberAddressList)  | Transaction receipt | Remove members from Data Union                                 |
+| setAdminFee(newFeeFraction[, ethersOptions])                                                      | Transaction receipt     | `newFeeFraction` is a `Number` between 0.0 and 1.0 (inclusive) |
```

Comment:
Update the new parameter signature also for `transportMessage` (line 416)

Reply:
not sure what this means since it already has the link to ethers docs where you can see those available ethers-options